app/Controllers/feedController.php

```diff
@@ -39,12 +39,14 @@ public function firstAction() {
 	 * @throws FreshRSS_Feed_Exception
 	 * @throws Minz_FileNotExistException
 	 */
-	public static function addFeed($url, $title = '', $cat_id = 0, $new_cat_name = '', $http_auth = '') {
+	public static function addFeed($param_url, $title = '', $cat_id = 0, $new_cat_name = '', $http_auth = '') {
```

Comment:
Please either keep `$url` (and later below use `$url = trim($url);`) or update the `@param` documentation a few lines higher up. I would better like option 1.

Reply:
I thought of writing something like that and decided against it without realizing that GitHub was hiding the documentation a few lines up. :-P

I'd also go with option 1.

Reply:
Did option 1 👍